### ISSUE TITLE
Big Summation and Product Operators Hard to Read in .tex and .html

### ISSUE THREAD

**Issue by njericha** [JacquesCarette/literate-scientific-software#273]
### Original Case Study Equation
![2017-06-15 2](https://user-images.githubusercontent.com/26230851/27193488-fffaa882-51cc-11e7-8476-f4ca008d15d6.png)

### Drasil .tex output
![2017-06-15](https://user-images.githubusercontent.com/26230851/27193493-02351a7e-51cd-11e7-875b-587c3ff07178.png)

### Drasil .html output
![2017-06-15 1](https://user-images.githubusercontent.com/26230851/27193496-05f2cd64-51cd-11e7-8768-c4e0b5e98534.png)

The tex and html versions of this equation are significantly harder to read partly due to the big operator bounds being beside rather than on top/bottom, and partly due to missing brackets in the .tex file which make it ambiguous what is being summed/multiplied. Can we implement forced brackets in `Expr`? Or is this an acceptable formatting of this equation?

Note: Ignore indices not matching with the operators since they should be fixed once #265 is done.

**Comment by smiths** [comment]
Hello @njericha .  This issue of brackets has come up before.  If I remember correctly, @szymczdm and @JacquesCarette have argued that brackets should be added automatically by the generator, not by the user.  The idea is that the user gives the equation, but Drasil decides how to format it.  We did however want to add a facility to allow information on logical groupings to be added to the equation information.  This is discussed in issue #4 and briefly in the following wiki:

https://github.com/JacquesCarette/literate-scientific-software/wiki/Explicit-Grouping-of-Expressions

**Comment by njericha** [comment]
@smiths Yes, I agree that brackets should be auto generated. It is being done for html but not tex currently. The wiki on groupings makes sense for specific contexts.

**Comment by smiths** [comment]
Thanks @njericha .  I hadn't noticed the difference between the tex and html.  I agree that the tex file equation should have the brackets auto generated.  Is that something we can fix @szymczdm ?

**Comment by JacquesCarette** [comment]
Also, the placement of the sub/super scripts is not so pleasing.  This definitely can (and should) be improved.

**Comment by njericha** [comment]
Possibly look into https://www.mathjax.org/ to display math in html cleaner.

**Comment by njericha** [comment]
![2017-06-19 1](https://user-images.githubusercontent.com/26230851/27288540-02f8ece6-54d5-11e7-840c-e8a9f9372ca1.png)
I would also like to add a point that the piecewise functions are also hard to read in the HTML version.

**Comment by JacquesCarette** [comment]
There should be no closing brace - why is it there?  Can you open a separate issue for this?

**Comment by njericha** [comment]
Yes @JacquesCarette see #283

**Comment by njericha** [comment]
Just to clarify, the above commit only fixes this in LaTeX, not HTML
